platform/libs.batik/external/binaries-list

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+033A3DE659273BA8B99346B2D005E6F0AB731393 org.apache.xmlgraphics:batik-anim:1.11
+5DD23E3D6194D27F4892BF1A310FD7909E6AC085 org.apache.xmlgraphics:batik-awt-util:1.11
+B8E47B2594DF81BDA8B7D9520C71036A4B1C8459 org.apache.xmlgraphics:batik-bridge:1.11
+7C302C2DB49975D4A5176BDDE843038443F96C2D org.apache.xmlgraphics:batik-constants:1.11
+7B71FBA5DF4992BA6EAD8176FC1C87BDFDF92D59 org.apache.xmlgraphics:batik-css:1.11
+F5FE16DC3F99DDBF3A894FF8DCFB5AC00418B2B2 org.apache.xmlgraphics:batik-dom:1.11
+220A4D9A63621FAFDF4AA68C51B0D13427F5712B org.apache.xmlgraphics:batik-ext:1.11
+E705137CF79F71F72516877FDC51B1E6FF33BAFF org.apache.xmlgraphics:batik-gvt:1.11
+65E3680B22AAD1D80308837D9B56B0A9F4125642 org.apache.xmlgraphics:batik-i18n:1.11
+52968FE3116133C4172F994B7A65AFDC5B002A1F org.apache.xmlgraphics:batik-parser:1.11
+F7284DABC28AAA7407BEF45F31DD204E58A35810 org.apache.xmlgraphics:batik-script:1.11
+E23C7F7AD43F3005F17CBB349AB2C5941F109D0B org.apache.xmlgraphics:batik-svg-dom:1.11
+9748F51CCBF27E4D608EB8E3FE0599238D886CEB org.apache.xmlgraphics:batik-util:1.11
+2169B4361C717C698F68399F7CAD438FB35DDA6C org.apache.xmlgraphics:batik-xml:1.11
+B90B6AC57CF27A2858EAA490D02BA7945D18CA7B commons-io:commons-io:1.3.1
```

Comment:
Did we really not have commons-io anywhere else? Surprising.

Reply:
We do have a wrapper module for commons-io in ide/o.apache.commons.io, but I was unsure whether it was OK to have a module in "platform" depend on a module in "ide". Do you know anything about this?

(Same question with commons-logging.)

Reply:
(And the current build failure will be fixed once this question is resolved--it's indeed complaining about duplicate JARs.)

Reply:
hm, I wonder what happens if we move o.apache.commons.io to the platform cluster?

Reply:
BTW, the commons-logging error says

> enterprise/web.struts/external/commons-logging-1.0.4.jar and platform/libs.batik/external/commons-logging-1.0.4.jar are identical

So we could perhaps add commons-logging to platform and add dependencies to it.

OTOH, seems like a minor conflict for commons-logging, we could also edit nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps like the error message tells us.

Reply:
matthiasblaesing settled this in his patch (see his comment), which I just applied. He did the first thing you suggested, which was to move the commons-io/commons-logging modules to the platform cluster. Then we can depend on them.